vault/core_metrics.go

```diff
@@ -301,6 +301,12 @@ func (c *Core) emitMetricsActiveNode(stopCh chan struct{}) {
 			c.activeEntityGaugeCollector,
 			"",
 		},
+		{
+			[]string{"policy", "available", "count"},
```

Comment:
The use of the word "available" in this context, concerning the number of policies that exist, doesn't make sense.

Reply:
This is how policies installed in Vault and can be listed have been referred to internally, see for example: https://github.com/hashicorp/vault/blob/a652145fe889a3670e29e0ceebb74179935b8276/vault/policy_store.go#L600-L601

Reply:
@biazmoreira and I chatted about this and we concluded that it would be clearer as `configured`. 

We do use that language in our [public facing documentation](https://developer.hashicorp.com/vault/api-docs/system/policy#list-policies) so hopefully this will make it easier for users to understand its intent.